django/db/backends/postgresql/introspection.py

```diff
@@ -44,10 +44,10 @@ def get_table_list(self, cursor):
             SELECT c.relname, c.relkind
             FROM pg_catalog.pg_class c
             LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
-            WHERE c.relkind IN ('r', 'v')
+            WHERE c.relkind IN ('r', 'v', 't')
```

Comment:
I think that you meant to put `'f'` here? Please also sort the values alphabetically.

Reply:
you're right! I push the correct letter and sorted alphabetically!